mcp/deployment/src/test/java/io/quarkiverse/langchain4j/mcp/test/McpWithResourcesAsToolsTest.java

```diff
@@ -0,0 +1,56 @@
+package io.quarkiverse.langchain4j.mcp.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIterable;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import jakarta.inject.Inject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import dev.langchain4j.agent.tool.ToolSpecification;
+import dev.langchain4j.service.tool.ToolProvider;
+import dev.langchain4j.service.tool.ToolProviderResult;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class McpWithResourcesAsToolsTest {
```

Comment:
FYI there is https://github.com/langchain4j/langchain4j/blob/main/langchain4j-mcp/src/test/java/dev/langchain4j/mcp/client/integration/McpResourcesAsToolsTestBase.java that I would like to copy (and adapt) to the quarkus-langchain4j extension. But I can just take care of that later myself.